src/app/controllers/readonly_view_controller.ts

```diff
@@ -8,12 +8,19 @@ class ReadonlyViewController {
               private drawer:ContentBlockDrawer) {
   }
 
-  loadContent(dom:DomContentBlock) {
+  loadContent(dom:DomContentBlock, enableCaching:boolean = true) {
+    if(enableCaching) this.loadCachedContent(dom);
```

Comment:
is there a case where enableCaching = false? if not, remove the option and the if